osf_tests/test_search_views.py

```diff
@@ -95,6 +95,80 @@ def test_search_projects(self):
         res = self.app.get(url, {'q': self.project.title})
         assert_equal(res.status_code, 200)
 
+    def test_search_user(self):
+
+        url = '/api/v1/search/user/'
```

Comment:
@alexschiller `api_url_for('search_contributor')` returns `/api/v1/user/search/` not `/api/v1/search/user/`. The corresponding requests return different results.